### PR TITLE
jobs

### PR DESCRIPTION
**Pull request by Cvetomird91**
Created a new pull request and branch because of merging issues on the previous one.

### CODE REVIEW

**Comment by igorshubovych** [comment]
Original PR #556

**Comment by igorshubovych** [comment]
LGTM

File: pages/linux/jobs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- view jobs spawned by the current shell
+
+`jobs`
+
+- list jobs and their process ids
+
+`jobs -l`
+
+- display information about jobs witch changed status
```
Comment (felixonmars) [inline pages/linux/jobs.md]: There is a typo here: witch -> with :)

Comment (Cvetomird91) [inline pages/linux/jobs.md]: Fixed.

**Comment by felixonmars** [comment]
:+1: Thanks!

**Pull request merged**